### PR TITLE
Add details about SRV record in user guide

### PR DESCRIPTION
**Pull request by aswinsuryan**
Signed-off-by: Aswin Surayanarayanan <asuryana@redhat.com>

### CODE REVIEW

File: src/content/operations/usage/_index.en.md [hunk 1]
```diff
@@ -244,17 +244,44 @@ $ kubectl config use-context cluster3
 Switched to context "cluster3".
 ```
 
-The following commands create an `nginx` Service in the nginx-test namespace which targets TCP port 8080 on any Pod with the `app: nginx`
-label and exposes it on an abstracted Service port. When created, the Service is assigned a unique IP address (also called ClusterIP):
+The following commands create an `nginx` Service in the nginx-test namespace which targets TCP port 8080, with name http, on any Pod with the
```
Comment (dfarrell07) [inline src/content/operations/usage/_index.en.md]:
```suggestion
The following commands create an `nginx` Service in the `nginx-test` namespace which targets TCP port 8080, with name http, on any Pod with the
```

Namespaces should be in `code style`.

https://kubernetes.io/docs/contribute/style/style-guide/#use-code-style-for-object-field-names-and-namespaces

File: src/content/operations/usage/_index.en.md [hunk 2]
```diff
@@ -244,17 +244,44 @@ $ kubectl config use-context cluster3
...
+`app: nginx` label and exposes it on an abstracted Service port. When created, the Service is assigned a unique IP address
+(also called ClusterIP):
```
Comment (dfarrell07) [inline src/content/operations/usage/_index.en.md]:
```suggestion
(also called ClusterIP):
```

Is this an API object? If so, it should be `code style`.

https://kubernetes.io/docs/contribute/style/style-guide/#code-style-inline-code

File: src/content/operations/usage/_index.en.md [hunk 3]
```diff
@@ -244,17 +244,44 @@ $ kubectl config use-context cluster3
...
 
 ```bash
 $ kubectl create namespace nginx-test
 namespace/nginx-test created
 
 $ kubectl -n nginx-test create deployment nginx --image=nginxinc/nginx-unprivileged:stable-alpine
 deployment.apps/nginx created
+```
+
+`kubectl apply` the following yaml within the nginx-test namespace to create the service:
```
Comment (dfarrell07) [inline src/content/operations/usage/_index.en.md]:
```suggestion
`kubectl apply` the following YAML within the `nginx-test` namespace to create the service:
```

File: src/content/operations/usage/_index.en.md [hunk 4]
```diff
@@ -409,8 +461,34 @@ To test this, we will deploy the same `nginx` Service in the same namespace on *
 ```bash
 $ kubectl -n nginx-test create deployment nginx --image=nginxinc/nginx-unprivileged:stable-alpine
 deployment.apps/nginx created
+```
+
+`kubectl apply` the following yaml within the nginx-test namespace to create the service:
```
Comment (dfarrell07) [inline src/content/operations/usage/_index.en.md]:
```suggestion
`kubectl apply` the following YAML within the `nginx-test` namespace to create the service:
```

**Review by dfarrell07 — CHANGES_REQUESTED** [review]
Looks good, a few style nits inline.

**Review by tpantelis — APPROVED** [review]

**Review by dfarrell07 — APPROVED** [review]

**Pull request merged**